### DISCUSSION TITLE
Using OpenAI embeddings for Bertopic

### DISCUSSION THREAD

**Discussion by kaushikusc** [MaartenGr/BERTopic#1611] [Q&A]
I had a quick question about openai.Embedding.Create function which creates open AI embedding, using the OpenAIBackend in Bertopic. Given the open ai function (openai.Embedding.Create) now requires 'engine' as input instead of 'model' - how can I run this without getting an error from open AI where the error is requesting an engine/ deployment_id. Is there an earlier openai package/ version, that can still work with the model parameter and Bertopic or does this require a code change?

**Comment by MaartenGr** [comment]
That will be possible in the upcoming release. There is already a PR for that if you want to try it out in #1572
